src/staging/src/styles/parks.scss

```diff
@@ -17,20 +17,20 @@ $mediumBreakpoint: 991.98px;
 }
 
 .heading {
-  color: #000;
+  color: $colorBlue;
 }
```

Comment:
I think you can delete this class after removing it from ~/staging/src/components/park/heading.js